wrappers/opencv/grabcuts/rs-grabcuts.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include <librealsense2/rs.hpp> // Include RealSense Cross Platform API
 #include <opencv2/opencv.hpp>   // Include OpenCV API
+#include <opencv2/highgui/highgui_c.h>
```

Comment:
C API is deprecated. Better replace everywhere
cvGetWindowHandle(window_name)
with
getWindowProperty(window_name, WND_PROP_AUTOSIZE) >= 0
to avoid future incompatibilities